examples/application_commands/autocompleted_command.py

```diff
@@ -0,0 +1,33 @@
+from nextcord import Interaction, SlashOption
+from nextcord.ext import commands
+
+bot = commands.Bot(command_prefix="/")
```

Comment:
again why not client, as it looks here like the command prefix is needed for slash